src/doom/g_game.c

```diff
@@ -1306,9 +1306,9 @@ void G_ScreenShot (void)
 int pars[4][10] = 
 { 
     {0}, 
-    if (gameversion == exe_chex && gamemap < 6)
+    if (gameversion == exe_chex)
```

Comment:
Sorry, but this isn't even valid C code. You cannot have an if-condition in an initialization.